src/llmtuner/webui/chatter.py

```diff
@@ -115,7 +116,7 @@ def predict(
         temperature: float,
     ) -> Generator[Tuple[Sequence[Tuple[str, str]], Sequence[Tuple[str, str]]], None, None]:
         chatbot.append([query, ""])
-        query_messages = messages + [{"role": Role.USER.value, "content": query}]
+        query_messages = messages + [{"role": Role(role), "content": query}]
```

Comment:
Shouldn't it be `Role(role).value` or just `role`?

Reply:
just role should be fine